website/docs/plugins/ax_sweeper.md

```diff
@@ -81,10 +81,12 @@ In general, the plugin supports setting all the Ax supported [Parameters](https:
 * `type` - Type of the parameter. It can take the following values: `range`, `fixed`, or `choice`.
 * `bounds` - Required only for the `range` parameters. It should be a list of two values, with the lower bound first.
 * `values` - Required only for the `choice` parameters. It should be a list of values.
-* `value` - Required only for the `fixed` parameters. It should be a single value. 
+* `value` - Required only for the `fixed` parameters. It should be a single value.
 
 Note that if you want to sample integers in the range `-5` to `5`, you need to specify the range as `int(interval(-5, 5))` (in the command line) or `[-5, 5]` (in config). If you want to sample floats in range `-5` to `5`, you need to specify the range as `interval(-5, 5)` (in the command line) or `[-5.0, 5.0]` (in config).
 
+Generally, Ax expects the function being optimized to return a tuple of `(measurement_value, measurement_uncertainty)`. If the return value is instead a scalar, Ax by default treats the uncertainty as an unknown and infers its value as the experiment progresses. The example above uses an exact function, so the `is_exact` parameter is set to `true`. As a result, the uncertainty of each scalar measurement is set to 0. Note that returning a `(measurement_value, measurement_uncertainty)` tuple will override this behavior.
```

Comment:
``is_exact` parameter is set to `true`` -> is this referencing the change in the example config? probably better to override and show in the docs (eg here `python example/banana.py -m 'banana.x=int(interval(-5, 5))' 'banana.y=interval(-5, 10.1)'`)